web/src/containers/App.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Counter from '../components/Counter';
+import { decrement, increment } from '../actions';
+
+const App = ({ dispatch, total }) => (
+  <div>
+    <Counter
+      total={total}
+      onIncrement={() => dispatch(increment())}
+      onDecrement={() => dispatch(decrement())}
```

Comment:
I'm down with this, especially for something so simple.  How do you feel about the `mapDispatchToProps` approach for more complicated dispatch events?

Reply:
yep, def prefer `mapDispatchToProps` (as well as `bindActionCreators`) once we start going for realsies